oppia/oppia-android
# Fix #1460 & #1249: Update topic calculation and introduced new json files.

## Explanation


Fix #1460 & #1249
This PR finishes the work for #1460 .

Changes done:
1. Removed `TOPIC_FILES_ASSOCIATION` because we need to make this dynamic and therefore optimised code to calculate it dymanically.
2. While calculating the topic size, there was crash because we did not have json files for all subtopics and therefore introduced 10 new json files for subtopics. Also, one issue that I noticed is that subtopics start with increment-numeric ids like `1`, `2`, `3`, etc. Now, currently its not an issue but if we happen subtopics for two different topics, this will be an issue and therefore updated the file names to `<topic_id>_<subtopic_id>.json`.
3. In `questions.json`, renamed `questions` -> to `question_dicts` 
4. Also added empty `thumbnail_filename` and `thumbnail_icon_url` which will be needed in future PRs.
5. Check First subtopic in `Fractions` topic. Also this will contain extra data for better explanation of revision card.

## How to test:
Just notice the topic size for all topics before checking out in this PR (which you can also see in below screenshots) and when you run this the size will remain same proving that algorithm will was written correctly.

## Screenshots:
![Screenshot_1595451029](https://user-images.githubusercontent.com/9396084/88232755-089db200-cc94-11ea-9c8d-287fc89095de.png)
![Screenshot_1595451034](https://user-images.githubusercontent.com/9396084/88232760-0b000c00-cc94-11ea-8ac8-a8970606849a.png)
![Screenshot_1595451038](https://user-images.githubusercontent.com/9396084/88232761-0c313900-cc94-11ea-9978-99e3dc097e94.png)
![Screenshot_1595451042](https://user-images.githubusercontent.com/9396084/88232764-0cc9cf80-cc94-11ea-8cde-2bf0e322f2d3.png)

## Checklist
